src/sendalert.cpp

```diff
@@ -86,13 +86,19 @@ void ThreadSendAlert()
     //  2000 for longer invalid proof-of-work chain 
     //  Higher numbers mean higher priority
     //  4000 or higher will put the RPC into safe mode
-    alert.nPriority     = 5000;
+    alert.nPriority     = 1500;
     alert.strComment    = "";
-    alert.strStatusBar  = "URGENT: Upgrade required: see https://z.cash";
-    alert.strRPCError   = "URGENT: Upgrade required: see https://z.cash";
+    alert.strStatusBar  = "Your client is out of date and potentially vulnerable to denial of service. Please update to the most recent version of Zcash (1.0.5). More info at: https://z.cash/support/security.html";
```

Comment:
should be "1.0.6", maybe "1.0.7", why not using zcash VERSION already defined to keep it dynamic!!